deploy/utils.sh

```diff
@@ -162,40 +158,48 @@ function deploy_test_env {
   export CONJUR_APPLIANCE_URL=$conjur_appliance_url
   export CONJUR_AUTHN_URL=$conjur_authenticator_url
 
-  echo "Deploying test-env"
-  wait_for_it 600 "$TEST_CASES_DIR/test-env.sh.yml | $cli_without_timeout apply -f -"
+  echo "Running Deployment Manifest"
 
-  expected_num_replicas=`$TEST_CASES_DIR/test-env.sh.yml |  awk '/replicas:/ {print $2}' `
+  if [[ "$DEV" = "true" ]]; then
+    ../dev/config/k8s/init-solution.sh.yml > ../dev/config/k8s/init-solution.yml
+    $cli_with_timeout apply -f ../dev/config/k8s/init-solution.yml
```

Comment:
```suggestion
    ./dev/config/k8s/init-solution.sh.yml > ./dev/config/k8s/init-solution.yml
    $cli_with_timeout apply -f ./dev/config/k8s/init-solution.yml
```

this worked for me locally